tools/packaging/common/iptables-retry.diff

```diff
@@ -0,0 +1,108 @@
+diff --git a/tools/packaging/common/istio-iptables.sh b/tools/packaging/common/istio-iptables.sh
+index 1bb70f3a..77629371 100755
+--- a/tools/packaging/common/istio-iptables.sh
++++ b/tools/packaging/common/istio-iptables.sh
+@@ -116,6 +116,77 @@ function isIPv6() {
+   done
+   return 0
+ }
++#
++# Maximum time to wait for an iptables call to succeed when populating the redirection rules.
++# During that time, iptables call will be attempted for every one second.
++#
++IPTABLES_WAIT=30
++#
++# The path to the iptables command.
++#
++# shellcheck disable=SC2230
++IPTABLES_CMD=$(which iptables)
++#
++# The path to the ip6tables command.
++#
++# shellcheck disable=SC2230
++IP6TABLES_CMD=$(which ip6tables)
++#
++# If true, retries iptables & ip6tables commands in case they fails.
++#
++IPTABLES_RETRY=false
++#
++# Function wrapping iptables to retry in case of failure if IPTABLES_RETRY is "true".
++#
++function iptables {
++    if [ "${IPTABLES_RETRY}" != "true" ]; then
++        "$IPTABLES_CMD" "$@"
++        return $?
++    fi
++
++    local iptables_wait=$IPTABLES_WAIT
++    set +o errexit
++    while true; do
++        "$IPTABLES_CMD" "$@"
++        local err_code=$?
++        if [[ $err_code == 0 ]]; then
++            break
++        fi
++        iptables_wait=$(( iptables_wait - 1 ))
++        if (( iptables_wait > 0 )); then
++            sleep 1
++        else
++            return $err_code
++        fi
++    done
++    set -o errexit
++}
++#
++# Function wrapping ip6tables to retry in case of failure if IPTABLES_RETRY is "true".
++#
++function ip6tables {
```

Comment:
see how I did this in #100.  I prefer that approach to consistent retry logic for both ipv4/v6.  Also, the ip6tables cleanup needs to be before this and, IMHO, should be unconditional like the v4 chains cleanup.

Reply:
@tiswanso Agreed. Just merge #100 and I'll update to pick that up.

Reply:
@tiswanso I picked up the change to unconditionally cleanup the `ip6tables` tables. Please review.